packages/main/src/Toolbar.hbs

```diff
@@ -0,0 +1,22 @@
+<div
+	class="ui5-tb-root"
+	tabindex="{{tabIndex}}"
+	part="otb"
+>
+
+	<div class="{{classes.actions}}" @click="{{onCustomActionClick}}">
+		{{#each standardActions}}
+			{{this.toolbarTemplate}}
+		{{/each}}
+
+		{{#if showOverflowBtn}}
+			<ui5-button
+				icon="overflow"
+				design="Transparent"
+				?disabled="{{this.disabled}}"
+				@click="{{onBtnOverflowClick}}"
```

Comment:
There is a `@click` here, and another `@click` on the div above. When you click a `ui5-button`, it will bubble (unless you stop it, I haven't checked for that yet), so do we need 2 event handlers?